src/app/manifest/manifest-service-json.ts

```diff
@@ -319,9 +335,15 @@ async function saveManifestToIndexedDB(
   }
 }
 
-function deleteManifestFile() {
+async function deleteManifestFile() {
   localStorage.removeItem(localStorageKey);
-  return del(idbKey);
+  await Promise.all(
+    (await keys()).map(async (key) => {
+      if (typeof key === 'string' && key.startsWith(idbKey)) {
+        await del(key);
```

Comment:
```suggestion
    (await keys()).map((key) => {
      if (typeof key === 'string' && key.startsWith(idbKey)) {
        return del(key);
```
?

Reply:
just seems like some keyword overhead to ultimately convert a promise to a promise. there's no followup work to the promise unlike loadManifestFromCache